browser/ui/webui/brave_new_tab_message_handler.cc

```diff
@@ -102,6 +102,9 @@ base::DictionaryValue GetPreferencesDictionary(PrefService* prefs) {
   pref_data.SetBoolean(
       "showGemini",
       prefs->GetBoolean(kNewTabPageShowGemini));
+  pref_data.SetBoolean(
```

Comment:
Should be within `#if BUILDFLAG(BITCOIN_DOT_COM_ENABLED)`

Reply:
fixed